src/open_producten/producttypen/serializers/jsonschema.py

```diff
@@ -0,0 +1,21 @@
+from django_json_schema_model.models import JsonSchema
+from jsonschema import Draft202012Validator
+from jsonschema.exceptions import SchemaError
+from rest_framework import serializers
+
+
+class JsonSchemaSerializer(serializers.ModelSerializer):
+
+    schema = serializers.DictField()
```

Comment:
Why is this line needed? Is including it in `Meta` not enough?

Reply:
Spectacular will render the type of JsonField as `any` in the type of `dictField` as `object`.

Reply:
Specifying `DictField` just for drf-spectacular feels awkward and doesn't solve it for all JSON fields (like new field `product.verbruiksobject`), I think some custom drf-spectacular field extension would be a better solution here (something like this - https://github.com/tfranzel/drf-spectacular/issues/1242)